modules/installation-vsphere-config-yaml.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere.adoc
+
+[id="installation-vsphere-config-yaml-{context}"]
+= Sample `install-config.yaml` file for vSphere
+
+You can customize the `install-config.yaml` file to specify more details about
+your {product-title} cluster's platform or modify the values of the required
+parameters.
+
+[source,yaml]
+----
+apiVersion: v1beta4
+baseDomain: example.com <1>
+compute:
```

Comment:
The `compute` field is neither used not needed for a vSphere install.

Reply:
@staebler 
About the replicas in install-config.yaml, I noticed only master field shown in the doc here.
1. Then how installer decide the number of computer nodes?
2. And how installer decide the number of master nodes?just respect to replicas in install-config.yaml?
I found when the replicas in install-config is not equal with the number of hosts created by terraform, the master node number will respect to the actual hosts number. Is this expected?
```
# cat .openshift_install_state.json |grep replicas
                "replicas": 3,
                    "replicas": 3,
# cat terraform.tfvars|grep control_plane_count
control_plane_count = 4
// must match the value of control_plane_count.
# oc get node
NAME              STATUS                     ROLES    AGE   VERSION
compute-0         Ready                      worker   15m   v1.13.4+27a00af64
compute-1         Ready                      worker   15m   v1.13.4+27a00af64
compute-2         Ready                      worker   16m   v1.13.4+27a00af64
control-plane-0   Ready                      master   15m   v1.13.4+27a00af64
control-plane-1   Ready,SchedulingDisabled   master   15m   v1.13.4+0e917197e
control-plane-2   Ready,SchedulingDisabled   master   15m   v1.13.4+0e917197e
control-plane-3   Ready                      master   15m   v1.13.4+27a00af64
```

Reply:
>     1. Then how installer decide the number of computer nodes?
The installer does not decide the number of compute nodes. The end user decides that. The installer does not care how many compute nodes there are. The installer is not created the compute nodes.

>     2. And how installer decide the number of master nodes?just respect to replicas in install-config.yaml?
The user is required to specify the number of control plane nodes in the install-config.yaml. This is required so that the appropriate Endpoints can be created for etcd.

Reply:
To be more clear, I want to ask, even end user "decide" the number of control plane nodes and create all the hosts, if that means, installer will according to actual hosts to schedule the control plane to the master machines, but not the replicas in config. The replicas just decide the number of etcd endpoints. Is this right?

Reply:
Yes, for UPI the replicas only dictate the number of etcd endpoints.

Reply:
@kalexand-rh 
Thx for @staebler's confirm. Here suggest to give a brief explanation the "replicas" in install-config sample [1], that "the number of etcd endpoints", which is not quite the same with the config in ipi.
[1] http://file.rdu.redhat.com/kalexand/0418/osdocs324/installing/installing_vsphere/installing-vsphere.html#installation-vsphere-config-yaml-installing-vsphere

Reply:
I think that would only confuse things. The user *should* be setting the replicas to the number of control plane machines that they will be creating.